kernelci/cli/user.py

```diff
@@ -49,6 +50,54 @@ def _api_call(self, api, configs, args):
         return True
 
 
+class cmd_get_group(APICommand):  # pylint: disable=invalid-name
+    """Get a user group with a given ID"""
+    args = APICommand.args + [Args.group_id]
+    opt_args = APICommand.opt_args + [Args.indent]
+
+    def _api_call(self, api, configs, args):
+        group = api.get_group(args.group_id)
+        self._print_json(group, args.indent)
+        return True
+
+
+class cmd_find_groups(AttributesCommand):  # pylint: disable=invalid-name
+    """Find user groups with arbitrary attributes"""
+    opt_args = AttributesCommand.opt_args + [
+        Args.limit, Args.offset
+    ]
+
+    def _api_call(self, api, configs, args):
+        attributes = self._split_attributes(args.attributes)
+        groups = api.get_groups(attributes, args.offset, args.limit)
+        self._print_json(groups, args.indent)
+        return True
+
+
+class cmd_get_user(APICommand):  # pylint: disable=invalid-name
+    """Get a user profile with a given profile ID"""
+    args = APICommand.args + [Args.user_profile_id]
+    opt_args = APICommand.opt_args + [Args.indent]
+
+    def _api_call(self, api, configs, args):
+        profile = api.get_user_profile(args.user_profile_id)
+        self._print_json(profile, args.indent)
+        return True
+
+
+class cmd_find_users(AttributesCommand):  # pylint: disable=invalid-name
+    """Find user profiles with arbitrary attributes"""
+    opt_args = AttributesCommand.opt_args + [
+       Args.limit, Args.offset
+    ]
+
+    def _api_call(self, api, configs, args):
+        attributes = self._split_attributes(args.attributes)
+        profiles = api.get_user_profiles(attributes, args.offset, args.limit)
+        self._print_json(profiles, args.indent)
+        return True
+
+
```

Comment:
Likewise, I think we can leave this out for now.  There's already the `whoami` command to get details about the current user.

Reply:
Removed `get` command.